Neos.Eel/Classes/Package.php

```diff
@@ -21,22 +21,24 @@
  */
 class Package extends BasePackage
 {
-    const EelExpressionRecognizer = '/
-			^\${(?P<exp>
-				(?:
-					{ (?P>exp) }			# match object literal expression recursively
-					|[^{}"\']+				# simple eel expression without quoted strings
-					|"[^"\\\\]*				# double quoted strings with possibly escaped double quotes
-						(?:
-							\\\\.			# escaped character (quote)
-							[^"\\\\]*		# unrolled loop following Jeffrey E.F. Friedl
-						)*"
-					|\'[^\'\\\\]*			# single quoted strings with possibly escaped single quotes
-						(?:
-							\\\\.			# escaped character (quote)
-							[^\'\\\\]*		# unrolled loop following Jeffrey E.F. Friedl
-						)*\'
-				)*
-			)}
-			$/x';
+    public const EelExpressionRecognizer = <<<'REGEX'
```

Comment:
actual modifications:

- use nowdoc (so we dont have to escape backslashes)
- adjust identation
- add atomic group `(?>`